Nightly testing fails with LNK1201 and LNK1318 on Windows x86

Example of failed run: https://github.com/ispc/ispc/actions/runs/26149953929
```
LINK : the 32-bit linker (C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.44.35207\bin\HostX86\x86\link.exe) ran out of heap space and is going to restart linking with a 64-bit linker
LINK : restarting link with 64-bit linker `C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.44.35207\bin\HostX64\x86\link.exe'
LINK : fatal error LNK1201: error writing to program database 'D:\a\ispc\ispc\tmp940\varying-struct-3.ispc.pdb'; check for insufficient disk space, invalid path, or insufficient privilege
Compilation of test .\tests\func-tests\varying-struct.ispc failed            
test_static.cpp
libvcruntime.lib(chandler4.obj) : fatal error LNK1318: Unexpected PDB error; RPC (23) '(0x000000A4)'
Compilation of test .\tests\func-tests\varying-struct-8.ispc failed            
test_static.cpp
libucrt.lib(invalid_parameter.obj) : fatal error LNK1318: Unexpected PDB error; RPC (23) ''
Compilation of test .\tests\func-tests\varying-struct-9.ispc failed            
test_static.cpp
LINK : fatal error LNK1318: Unexpected PDB error; RPC (23) ''
```

--- a/.github/workflows/scripts/load-vs-env.ps1
+++ b/.github/workflows/scripts/load-vs-env.ps1
@@ -1,9 +1,13 @@
 if($args[0] -ceq "x86") {
     $arch="x86"
+    # Use 64-bit hosted tools to avoid 32-bit linker memory limitations (LNK1201, LNK1318)
+    $host_arch="x64"
 } elseif($args[0] -ceq "x86-64") {
     $arch="x64"
+    $host_arch="x64"
 } elseif($args[0] -ceq "aarch64") {
     $arch="arm64"
+    $host_arch="arm64"
 } else {
 	Write-Host "Unrecognized architecture - use of these: x86, x86-64."
 	Exit 1
@@ -21,4 +25,4 @@ Write-Output "  <-> VS Install Path: ${VS_INST_PATH}"
 
 # Import DevShell module and enter the dev shell environment
 Import-Module ${VS_INST_PATH}/Common7/Tools/Microsoft.VisualStudio.DevShell.dll
-Enter-VsDevShell -VsInstallPath ${VS_INST_PATH} -SkipAutomaticLocation -DevCmdArguments "-arch=${arch} -no_logo"
+Enter-VsDevShell -VsInstallPath ${VS_INST_PATH} -SkipAutomaticLocation -DevCmdArguments "-arch=${arch} -host_arch=${host_arch} -no_logo"

--- a/.github/workflows/scripts/load-vs-env.ps1
+++ b/.github/workflows/scripts/load-vs-env.ps1
@@ -9,7 +9,7 @@ if($args[0] -ceq "x86") {
     $arch="arm64"
     $host_arch="arm64"
 } else {
-	Write-Host "Unrecognized architecture - use of these: x86, x86-64."
+	Write-Host "Unrecognized architecture - use one of these: x86, x86-64, aarch64."
 	Exit 1
 }
 

